checkstyle.xml

```diff
@@ -164,12 +164,6 @@
         </module>
         <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance"/>
-        <module name="CustomImportOrder">
```

Comment:
Not opposed, but why remove this?

Reply:
I removed this because I just couldn't otherwise figure out how to get the checkstyle plugin to calm down.  I tried changing the settings on IntelliJ's import ordering so that the program would automatically perform the reorderings across all the files to no avail, and also tried manual reorderings and couldn't satisfy the checkstyler.  At a certain point I had wasted way too much time on it.  Maybe we could solve this in a separate PR dedicated specifically to getting the import settings worked out how we want them?  I'm sure there's a happy middleground that gets us some alphabetizing without being overly strict.

Reply:
Would be great to follow up. 

I believe what this is saying is create groups that are:

`static import`

special groups (org.interledger) - this is the wrong value and is a valuable lesson in stealing configs from other projects

third party packages - not sure

standard java packages. 

Each group is broken out by a vertical space and in each group imports are alphabetized. This isn't a huge deal but if that clears things up and we can avoid removing this rule that would be great.

Reply:
Ok great, I've assigned myself an Asana task for this next sprint.